src/types/events-types.ts

```diff
@@ -60,7 +38,7 @@ export type EventsQueryDescriptor = {
   interface: DwnInterfaceName.Events;
   method: DwnMethodName.Query;
   messageTimestamp: string;
-  filters?: EventsFilter[];
+  filters: EventsFilter[];
```

Comment:
Unrelated to this PR, but didn't like the duality between an undefined filter and empty array, so made filters required and allowed an empty array in the message descriptor.